### DISCUSSION TITLE
App code source availibility

### DISCUSSION THREAD

**Discussion by farfromrefug** [community/community#18421] [Mobile]
Do you plan on making the app open source? It would be great to be able to have it.
You did some really great things with it and it would be awesome for everyone to benefit from it.

Hope you'll agree

Thanks

**Comment by Zerotask** [comment]
Don't think so. GitHub itself is closed source.

If you're interested in mobile app development, I can recommend you trying Flutter

**Reply by farfromrefug** [reply]
@Zerotask thanks but it is not a question of using flutter or any other framework. 
I hope the github team will review this as it nothing to with github IP.

**Reply by ghostofcodes** [reply]
If you are really looking for open source apps I guess you can just use the search bar and type `open source app`? I mean, GitHub is full of open source projects haha and that way, you can even sort results in the language of your choice
